package.json

```diff
@@ -11,14 +11,24 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "bootstrap": "^4.5.2",
+    "jquery": "^3.5.1",
+    "popper.js": "^1.16.1",
     "webpack": "^4.44.1",
     "webpack-cli": "^3.3.12"
   },
   "devDependencies": {
```

Comment:
- [ ] All webpack related packages should be included in package.json like webpack, webpack-cli, webpack-dev-server etc. Also, those packages should be added as devDependencies ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)).
Please move the required dependencies to the devDependencies section on the packeage.json file.